docs/pom.xml

```diff
@@ -57,7 +142,7 @@
                     <failOnMissingWebXml>false</failOnMissingWebXml>
                     <webResources>
                         <resource>
-                            <directory>target/_site/docs</directory>
+                            <directory>${project.build.directory}/_site</directory>
```

Comment:
@benoitf I think this is what I have done. I will change any `exec dir="${project.basedir}"` to `exec dir="${project.build.directory}"` to ensure this. Docker commands do no use `volumes` but uses instead `cp` so there is no chance for it to write to ${project.basedir}.